### PR TITLE
Migrate `eth` page

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Migrates the `eth` page to use Chakra UI.

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/9353

### CODE REVIEW

File: src/pages-conditional/eth.tsx [hunk 1]
```diff
@@ -16,153 +24,204 @@ import InfoBanner from "../components/InfoBanner"
...
+        flexDirection: { sm: "column" },
+        ml: { sm: 4 },
+      },
+      p: {
+        mb: { sm: 0 },
+      },
+    }}
+    {...props}
+  />
+)
```
Comment (minimalsm) [inline src/pages-conditional/eth.tsx:224]: Image placement on mobile res is a little low / not centred

image

File: src/pages-conditional/eth.tsx [hunk 2]
```diff
@@ -16,153 +24,204 @@ import InfoBanner from "../components/InfoBanner"
...
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    margin-top: 1.5rem;
-  }
-`
+const Title = (props: ChildOnlyProp) => (
+  <Heading
+    as="h1"
+    fontSize="sm"
+    lineHeight={1.4}
+    letterSpacing="wider"
```
[the remark this answers is not in the document]
Comment (pettinarip) [reply src/pages-conditional/eth.tsx]: Correct. Lets keep as close as possible to the current styles.

File: src/pages-conditional/eth.tsx [hunk 3]
```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
...
+)
 
-const StyledCardContainer = styled(CardContainer)`
-  margin-bottom: 2rem;
-`
+export const LeftColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="0 0 50%"
+    maxW={{ base: "full", lg: "75%" }}
+    mr={{ lg: 16 }}
```
Comment (wackerow) [inline src/pages-conditional/eth.tsx:124]: `me` for `margin-inline-end` (rtl responsive)
```suggestion
    me={{ lg: 16 }}
```

File: src/pages-conditional/eth.tsx [hunk 4]
```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
...
+    as="h4"
+    fontSize={{ base: "md", md: "xl" }}
+    fontWeight={600}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const CardContainer = (props: FlexProps) => (
+  <Flex wrap="wrap" ml={-4} mr={-4} {...props} />
```
Comment (wackerow) [inline src/pages-conditional/eth.tsx:203]:
```suggestion
  <Flex wrap="wrap" my={-4} {...props} />
```

File: src/pages-conditional/eth.tsx [hunk 5]
```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
...
-  }
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    display: flex;
+const Hero = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 100%"
+    maxW="800px"
+    bgSize="cover"
+    bgRepeat="no-repeat"
+    ml={{ base: 0, md: 8 }}
```
Comment (wackerow) [inline src/pages-conditional/eth.tsx:159]: `ms` for `margin-inline-start` (rtl responsive)
```suggestion
    ms={{ base: 0, md: 8 }}
```

**Review by wackerow — APPROVED** [review]

File: src/pages-conditional/eth.tsx (hunk 4 above)
Comment (pettinarip) [reply src/pages-conditional/eth.tsx]: @wackerow shouldn't have been `mx` instead? or why `my`?

Comment (wackerow) [reply src/pages-conditional/eth.tsx]: Oops, yes... should be `mx`

File: src/pages-conditional/eth.tsx [hunk 6]
```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
...
+    as="h4"
+    fontSize={{ base: "md", md: "xl" }}
+    fontWeight={600}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const CardContainer = (props: FlexProps) => (
+  <Flex wrap="wrap" my={-4} {...props} />
```
Comment (pettinarip) [inline src/pages-conditional/eth.tsx:203]:
```suggestion
  <Flex wrap="wrap" mx={-4} {...props} />
```

**Pull request merged**